### PR TITLE
0.17.0 `CHANGELOG`s

### PR DESCRIPTION
**Pull request by tiagobento**

### CODE REVIEW

File: packages/vscode-extension-bpmn-editor/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 0.17.0 (alpha)
```
Comment (domhanak) [inline packages/vscode-extension-bpmn-editor/CHANGELOG.md:1]: @tiagobento imho some features are missing here, for example the generic collections are now allowed in data types - KOGITO-6067

Comment (tiagobento) [reply packages/vscode-extension-bpmn-editor/CHANGELOG.md:1]: @domhanak KOGITO-6067 doesn't have a "Fix version/s" set, that's why it didn't appear on the release notes. If there are more like these, please update the Jiras and we'll fix the release notes.

**Review by ederign — APPROVED** [review]

File: packages/vscode-extension-bpmn-editor/CHANGELOG.md (hunk 1 above)
Comment (ederign) [reply packages/vscode-extension-bpmn-editor/CHANGELOG.md:1]: @tiagobento let's not hold the release due to that. We can always update the Github RN and also the blog post later.

Unfortunatelly, the VS Code RN cannot be updated after the release, but I'll talk again with the leads to make sure that the JIRAS has the properly fixed version.

Comment (domhanak) [reply packages/vscode-extension-bpmn-editor/CHANGELOG.md:1]: @tiagobento sure not blocking, just pointing out we miss a lot of them here from BPMN

**Review by domhanak — APPROVED** [review]

**Pull request merged**